src/OpenTelemetry.Exporter.Console/.publicApi/net462/PublicAPI.Shipped.txt

```diff
@@ -1,3 +1,5 @@
+*REMOVED*static OpenTelemetry.Logs.ConsoleExporterLoggingExtensions.AddConsoleExporter(this OpenTelemetry.Logs.OpenTelemetryLoggerOptions loggerOptions, System.Action<OpenTelemetry.Exporter.ConsoleExporterOptions> configure = null) -> OpenTelemetry.Logs.OpenTelemetryLoggerOptions
```

Comment:
what does *REMOVED* mean?

Reply:
The overload with an optional param was removed... _technically_ this could break someone who was calling this method reflectively or something.

Reply:
I guess these lines need to be manually removed from the Shipped files? I thought the tool understood *REMOVED* and would not have copied this to Shipped.

Reply:
Yeah it looks like we would have to remove it manually.

Reply:
Here is the commit which removes these entries: cfcf521f5b4a2d66f68627f55f396d8c00fab74f